cool-core/src/main/java/com/nus/cool/model/CoolModel.java

```diff
@@ -190,4 +199,21 @@ public File getCubeStorePath(String cube) throws IOException{
       return out;
 
   }
+
```

Comment:
Have you tested this resetCube?

Reply:
I have tested this function and have used this function in the server.

Reply:
But in the server, we have another idea that is to reload the CUBE every time when processing its queries.

Reply:
why do we need to do that?

Reply:
With this function, we do not need to reload the cube file every time when performing queries

Reply:
But we find that it would be better to reload all cube files in the distribution deployment.